internal/driver/glfw/window.go

```diff
@@ -29,22 +28,26 @@ const (
 )
 
 var (
-	defaultCursor, entryCursor, hyperlinkCursor *glfw.Cursor
-	initOnce                                    = &sync.Once{}
-	defaultTitle                                = "Fyne Application"
+	defaultCursor, textCursor, pointerCursor, crosshairCursor, hResizeCursor, vResizeCursor *glfw.Cursor
```

Comment:
can you move this to two lines please - otherwise it makes the following aligned items harder to read

Reply:
actually this probably doesn't matter, feel free to ignore that preference